pkg/csi/service/node.go

```diff
@@ -37,7 +37,11 @@ import (
 )
 
 const (
-	maxAllowedBlockVolumesPerNode = 59
+	// vCenter 8.0 supports attaching max 255 volumes to Node
+	// Previous vSphere releases supports attaching a max of 59 volumes to Node VM.
+	// Deployment YAML file for Node DaemonSet has ENV MAX_VOLUMES_PER_NODE set to 59 for vsphere-csi-node container
+	// If Customer is using vSphere 8.0, they are allowed to set MAX_VOLUMES_PER_NODE to 255
+	maxAllowedBlockVolumesPerNode = 255
```

Comment:
It would have been good if we could compare the VC version and then set this value but since its in the nodes, I guess that is not possible right?

Reply:
@sashrith 

ID 7 is still reserved.

https://user-images.githubusercontent.com/22985595/163840852-bdd50583-3c4d-4866-89ae-a873767017c4.mov

Reply:
@shalini-b I am not changing the default config max in the manifest YAML. Also on the vCenter, we are not enabling 256 disk support by default.

Only by following specific steps, user will be able to enable 256 disk support in vCenter and CSI. We can document the requirement.

Reply:
So, we should set it to 252 right!